packages/api-headless-cms/src/graphql/schema/contentModels.ts

```diff
@@ -123,6 +136,7 @@ export const createModelsSchema = (context: CmsContext): GraphQLSchemaPlugin<Cms
                 layout: [[ID!]!]
                 fields: [CmsContentModelFieldInput!]
                 titleFieldId: String
+                tags: [String]
```

Comment:
Add ! into the array. tag should not be null

Reply:
And now I see the create model mutation and it looks like tags must be an array and items in it must be string. So just add ! and make sure that null does not go out of the resolvers in any case.

Reply:
I'm setting this as `tags: [String!]`, because you don't have to send tags at all. If you don't send them, the API will still assign default tags for you, so from the GraphQL perspective, it's not required.